src/Symfony/Component/Routing/RouteCollection.php

```diff
@@ -294,4 +294,21 @@ public function addResource(ResourceInterface $resource)
             $this->resources[$key] = $resource;
         }
     }
+
+    /**
+     * Allow to filter a RouteCollection.
+     *
+     * @param \Closure $closure
+     *
+     * @return RouteCollection
+     */
+    public function filter(\Closure $closure)
```

Comment:
Does it worth it to add a new public method to maintain just for the command ? I guess the filtering could be made directly in the command.